Container Security/kernel-eks-implementation-guide.md

```diff
@@ -0,0 +1,242 @@
+   ```
+   $ helm upgrade --install falcon-helm crowdstrike/falcon-sensor \
+        -n falcon-system --create-namespace \
+        --set falcon.cid="$CID" \
+        --set node.image.repository=$FALCON_NODE_IMAGE_URI
+   Release "falcon-helm" does not exist. Installing it now.
+   NAME: falcon-helm
+   LAST DEPLOYED: Fri Mar  5 17:07:54 2021
+   NAMESPACE: falcon-system
+   STATUS: deployed
+   REVISION: 1
+   TEST SUITE: None
+   NOTES:
+   You should be a Crowdstrike customer and have access to the Falcon Linux Sensor
+   and Falcon Container Downloads to install this helm chart and have it work
+   correctly as a specialized container has to exist in the container registry
+   before this chart will install properly.
+
+   The CrowdStrike Falcon sensor should be spinning up on all your Kubernetes nodes
+   now. There should be no further action on your part unless no Falcon Sensor
+   container exists in your registry. If you forgot to add a Falcon Sensor image to
+   your image registry before you ran `helm install`, please add the Falcon Sensor
+   now; otherwise, pods will fail with errors and crash until there is a valid
+   image to pull. The default image name to deploy a kernel sensor to a node is
+   `falcon-node-sensor`.
+   ```
+   To learn more about falcon-helm visit [upstream github](https://github.com/CrowdStrike/falcon-helm).
+
+ - (optional) Verify that falcon-node-sensor pod running. You should see one pod per each node on your cluster.
+   ```
+   $ kubectl get pods -n falcon-system
+   NAME                              READY   STATUS    RESTARTS   AGE
+   falcon-helm-falcon-sensor-bs98m   2/2     Running   0          21s
+   ```
+ - (optional) Verify that given pod has registered with CrowdStrike Falcon and received unique identifier.
+   ```
+   $ kubectl exec -n falcon-system falcon-helm-falcon-sensor-bs98m -c falcon-node-sensor -- falconctl -g --aid
+   aid="a582XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+   ```
+ - (optional) Verify that Falcon Sensor for Linux has insert itself to the kernel
+    ```
+    $ lsmod | grep falcon
+    falcon_lsm_serviceable     724992  1
+    falcon_nf_netcontain        20480  1
+    falcon_kal                  45056  1 falcon_lsm_serviceable
+    falcon_lsm_pinned_11110     45056  1
+    falcon_lsm_pinned_11308     45056  1
+    ```
+
+## Uninstall helm chart
+
+ - Step 1: Uninstall the helm chart
+   ```
+   helm uninstall falcon-helm -n falcon-system
+   ```
+ - Step 2: Delete the falcon image from AWS ECR registry
+   ```
+   $ aws ecr batch-delete-image --region eu-west-1 \
+       --repository-name falcon-node-sensor \
+       --image-ids imageTag=latest
+   {
+       "imageIds": [
+           {
+               "imageDigest": "sha256:e14904d6fd47a8395304cd33a0d650c2b924f1241f0b3561ece8a515c87036df",
+               "imageTag": "latest"
+           }
+       ],
+       "failures": []
+   }
+   ```
```

Comment:
Where do we get the imageDigest value to copy/paste into this command?

Reply:
That looks like the json return value. Like mentioned above it would probably be good to distinguish command vs return values.